recipe/meta.yaml

```diff
@@ -1,15 +1,16 @@
-{% set version = "2.12.2" %}
+{% set version = "2.14.5" %}
 
 package:
   name: pylint
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pylint/pylint-{{ version }}.tar.gz
-  sha256: 9d945a73640e1fec07ee34b42f5669b770c759acd536ec7b16d7e4b87a9c9ff9
+  sha256: 487ce2192eee48211269a0e976421f334cf94de1806ca9d0a99449adcdf0285e
 
 build:
-  number: 1
+  number: 0
+  skip: True #[py<37]
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
   entry_points:
     - pylint = pylint:run_pylint
```

Comment:
Could you add please `pylint-config = pylint:_run_pylint_config` into `entry_points`? See https://github.com/PyCQA/pylint/blob/566ffca72a1a08c9e7ec14383c4710d64165c592/setup.cfg#L70